### ISSUE TITLE
Fantom programming language

### ISSUE THREAD

**Issue by lpvm** [slurpdurp/nogood#4]
It has lots of sweet features:
http://fantom.org/

**Comment by slurpdurp** [comment]
Yes, Fantom is very interesting and it's on the list of checked languages. Unfortunately, it didn't make the >= 10 active repos.
